src/main/java/homeworks/Infrastructure/utils/randomStringGenerator.java

```diff
@@ -0,0 +1,39 @@
+package homeworks.Infrastructure.utils;
+
+import java.util.Random;
+
+public class randomStringGenerator {
+
+     public enum typeEnum {
```

Comment:
rename according to Java name convention
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html